docs/eventing/samples/writing-ra-source/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Writing an event source from scratch"
```

Comment:
Let's update the title of this page and the folder name to be more easily readable and to match each other.

I'm guessing "ra" in the folder name is short for receive adapter? You could name the folder /writing-receive-adapter-source/ and the give the title "Writing an event source using a receive adapter" or something similar.

Or change the folder name to /writing-source-from-scratch/ and keep the title as it is. Either way, the title and file name should be closely linked.

Reply:
Changed to `writing-receive-adapter-source` as suggested, thanks